docs/references/nextjs/custom-signup-signin-pages.mdx

```diff
@@ -9,6 +9,9 @@ This guide shows you how to use the [`<SignIn />`](/docs/components/authenticati
 
 If Clerk's prebuilt components don't meet your specific needs or if you require more control over the logic, you can rebuild the existing Clerk flows using the Clerk API. For more information, check out the [custom flows](/docs/custom-flows/overview) guides.
 
+> [!NOTE]
+> Watch the video version of this guide on the Clerk YouTube channel  → [YouTube (4 minutes)](https://youtu.be/fsuHLafTYyg).
```

Comment:
@bookercodes can we remove the unicode arrow usage here? Doesn't look super great in use.